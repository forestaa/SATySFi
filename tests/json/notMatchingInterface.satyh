module X : sig
  val x : int
end = struct
  let x = 0.0
end
