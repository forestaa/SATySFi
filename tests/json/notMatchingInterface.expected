{
  "file": "notMatchingInterface.satyh",
  "range": {
    "start": { "line": 1, "character": 0 },
    "end": { "line": 5, "character": 3 }
  },
  "message":
    "! [Type Error] The implementation of value 'x' has type\n      float\n    which is inconsistent with the type required by the signature\n
 int"
}
